platform/common/core/generic/committer/finality.go

```diff
@@ -110,7 +110,7 @@ func (c *FinalityManager[V]) runStatusListener(ctx context.Context) {
 			c.logger.Debugf("check vault status for [%d] transactions", len(txIDs))
 			statuses, err := c.vault.Statuses(newCtx, txIDs...)
 			if err != nil {
-				c.logger.Errorf("error fetching statuses: %w", err)
+				c.logger.Errorf("error fetching statuses: %s", err.Error())
```

Comment:
please, Arne, while updating these logs, check also if a context.Context is available and in case, use the context logger. In this case, it would be ErrorfContext(ctx,...

This, in addition to writing a log, will also generate an event in the traces.
Please, make sure that the string printed is `printable`. Arguments might need to sanitized depending on the cases.